AUTHORS.md

```diff
@@ -0,0 +1,18 @@
+# Authors
+
+This example is based on the [original open data analysis](http://opendata.cern.ch/record/5500) by Jomhari, Nur Zulaiha; Geiser, Achim;
```

Comment:
Perhaps we should move the "this example is based..." after the author list, for better consistency with the other examples, and as a way of opening a room for putting an acknowledgements section elsewhere too.